agent/structs/config_entry_jwt_provider.go

```diff
@@ -245,6 +251,9 @@ type RemoteJWKS struct {
 	//
 	// There is no retry by default.
 	RetryPolicy *JWKSRetryPolicy `json:",omitempty" alias:"retry_policy"`
+
+	// JWKSCluster defines how the specified Remote JWKS "URI" is to be fetched.
```

Comment:
```suggestion
	// JWKSCluster defines how the specified Remote JWKS URI is to be fetched.
```

Or is there something that makes it different from a normal URI?

Reply:
i was trying to indicate the URI field in the remoteJWKS field but really doesn't matter that much. Updated